views/advanced_usage.dt

```diff
@@ -49,3 +49,10 @@ block body
 		dd You can specify arbitrary versions, branches, and paths in the <code>dub.selections.json</code> file, even if they contradict the dependency specification of the packages involved (note that DUB will output a warning in that case).
 
 	p Execute <code>dub &lt;command&gt; -h</code> for more information on these commands.
+
+	h2#subpkg Managing subpackages
+
+	p For adding subpackages in your project, execute <code> dub add &lt;packagename&gt;:&lt;subpackage&gt; </code>
```

Comment:
This reads to me like `dub add packagename:subpackage` would create a subpackage for me, but I don't think this is the case?